frontend/.env

```
@@ -1 +1,2 @@
-BASE_API_URL="https://uni-networking-app.onrender.com/api/v1"
\ No newline at end of file
+BASE_API_URL="https://uni-networking-app.onrender.com/api/v1"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)